_tutorials/bower-to-npm.md

```diff
@@ -28,15 +28,17 @@ _Note: These [drawbacks will be addressed with component updates by Q3 2021](/bl
 ## Migrating a project which uses Origami components from bower to npmjs
 
 ### TLDR
```

Comment:
Instead of having this page, what do we think about redirecting to the new migration guide? https://github.com/Financial-Times/origami-website/blob/cb595392b2a43dee22755589b277ed9a455afe69/_posts/2021-07-01-origami-on-npm-and-how-to-migrate.md#using-origami-via-bower